project/Dependencies.scala

```diff
@@ -20,8 +20,10 @@ object Dependencies {
   val scalaXmlVersion = "1.0.6"
   val aeronVersion = "1.15.1"
 
+  val scala212 = "2.12.8"
+
   val Versions = Seq(
-    crossScalaVersions := Seq("2.12.8", "2.11.12", "2.13.0-M5"),
+    crossScalaVersions := Seq(scala212, "2.11.12", "2.13.0-M5"),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
```

Comment:
FYI There's an additional complication, that `akka.build.scalaVersion` doesn't really work anymore, since it does not trigger sbt to filter the modules based on scala version, so for the relevant jenkins jobs we had to prefix every task with `++2.11.12`